echo-pubsub-core/src/main/java/com/netflix/spinnaker/echo/pubsub/utils/NodeIdentity.java

```diff
@@ -29,7 +29,7 @@
 
   private String runtimeName;
   private String hostName;
-  @Getter private String identity;
+  @Getter private String identity = UNKNOWN_HOST;
```

Comment:
I think this would be better in an `else` block in the constructor so that we're not setting this in two different places. Also you can make it `final` then. (And the other two fields can just be deleted, can't they? They're only used to calculate `identity`.)

Reply:
Good suggestions, applied.